mono/metadata/class-setup-vtable.c

```diff
@@ -1967,7 +1967,9 @@ mono_class_setup_vtable_general (MonoClass *klass, MonoMethod **overrides, int o
 		MonoMethod *impl = overrides [i*2 + 1];
 		if (!MONO_CLASS_IS_INTERFACE_INTERNAL (decl->klass)) {
 			g_assert (decl->slot != -1);
+#ifdef FEATURE_COVARIANT_RETURNS
```

Comment:
This seems unrelated, I'd rather move it out to another PR (and ping @lambdageek).

Reply:
It's just fixing an unused variable warning in mono/mono, I think.  I'm ok keeping it here

Reply:
Oh it's not a functional change? Gotcha.